packages/babel-helper-class-features-plugin/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@babel/plugin-class-features",
+  "name": "@babel/helper-class-features-plugin",
```

Comment:
maybe `@babel/helper-create-class-feature-plugin` or `@babel/helper-class-feature-factory`?

Reply:
I like the first name :+1:

Reply:
`@babel/helper-create-class-feature-plugin` 👍